src/app/shell/details/achievements/achievements.component.ts

```diff
@@ -13,30 +13,39 @@ import { UserState } from 'src/app/shared/store/user.state';
   templateUrl: './achievements.component.html',
   styleUrls: ['./achievements.component.scss'],
 })
-export class AchievementsComponent implements OnInit {
-  readonly noResultAchievements = NoResultsTitle.noAchievements;
-  @Input() achievements: Achievement[];
-
+export class AchievementsComponent implements OnInit {  
   @Select(UserState.achievements)
   achievements$: Observable<Achievement[]>;
-  @Input() workshop: Workshop;
+  @Select(UserState.selectedWorkshop) workshop$: Observable<Workshop>;
```

Comment:
move workshop$: Observable<Workshop>; to teh next row, so everything will be in the same styleCode